server/resolvers/book/mutation.js

```diff
@@ -0,0 +1,72 @@
+const { Author, Book } = require("../../models");
+const { Metadata, Review } = require("../../models/mongodb");
+const { sequelize } = require("../../models");
+
+const BookMutation = {
+  addBook: async ({ title, description, published_date, author_id }) => {
```

Comment:
As a reader, the function `addBook` is confusing, its not very clear what we are trying to do. anyway to give more clarity, breaking into functions for similar concern ?

Reply:
addBook tries to create a book, maybe `createBook` sounds better.